Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -400,6 +400,10 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _redirectToAttribute: function (attr) {
             var attrRedirectionUrl = this.get(attr);
 
+            this.set('publishRedirectionUrl');
```

Comment:
@dpobel 
1. the default value is `undefined` so I'm setting the default value. We set the redirection URLs when needed. There's no need to keep them after redirecting a user. Keeping it after user redirection makes the app behaviour unpredictable when someone wants to update it and redirect user somewhere else. If it still doesn't convinvce you, then what you propose to do in this case?
2. moving the reset of values into a separate method would work for you?